.github/workflows/tests.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     timeout-minutes: 10
     strategy:
+      fail-fast: false
```

Comment:
This is to allow maintainers to easily tell what versions of python may be affected by failing CI tests